assisted_compliance.md

```diff
@@ -0,0 +1,45 @@
+## Title
+Helping repo owners be compliant by writing their CONTRIBUTING.md for them as a pull request
+
+### Alternate titles
+* Let us help you be compliant
+* Assisted compliance
+
+## Problem
+The team that owns the repository doesn't have a CONTRIBUTING.md; the task force needs them to have this to submit bug fixes.
+
+## Context
+* Teams owning the repository resist compliance-related mandates for having a CONTRIBUTING.md
+* Compliance guys have to do a job; this is made difficult by teams resisting this.
+* Teams are focusing on business value; CONTRIBUTING.md compliance is viewed as not as important.
+* Special task force for security and compliance: developers responsible for fixing these bugs across the company.
+* You have to do these fixes.
```

Comment:
Can you elaborate on this one, @NewMexicoKid ?

Reply:
It's been a while, but I suspect that the "you have to do these fixes" line can be removed--it was probably a comment after the previous line.

Reply:
Fixing as suggested.